hack/verify-godeps.sh

```diff
@@ -102,14 +102,14 @@ pushd "${KUBE_ROOT}" 2>&1 > /dev/null
   fi
 
   if ! _out="$(diff -Naupr -x "BUILD" -x "OWNERS" -x "AUTHORS*" -x "CONTRIBUTORS*" vendor ${_kubetmp}/vendor)"; then
-    echo "Your vendored results are different:"
-    echo "${_out}"
-    echo "Godeps Verify failed."
+    echo "Your vendored results are different:" >&2
```

Comment:
could also wrap some of these long ones in a subshell, e.g.
```shell
(
  echo "blah"
  echo "blah"
) >&2
```
but this is fine too.